UnityProject/Assets/Scripts/PlayGroups/PlayerMove.cs

```diff
@@ -89,6 +90,25 @@ public Vector3Int GetNextPosition(Vector3Int currentPosition, PlayerAction actio
 			return currentPosition + adjustedDirection;
 		}
 
+		public string ChangeKeyboardInput(bool setAzerty)
+		{
+			ControlAction controlAction = UIManager.Action;
+			if (setAzerty)
+			{
+				keyCodes = new KeyCode[] { KeyCode.Z, KeyCode.Q, KeyCode.S, KeyCode.D, KeyCode.UpArrow, KeyCode.LeftArrow, KeyCode.DownArrow, KeyCode.RightArrow };
+				azerty = true;
+				controlAction.azerty = true;
+				PlayerPrefs.SetInt("AZERTY", 1);
+				PlayerPrefs.Save();
+				return "AZERTY";
+			}
+			keyCodes = new KeyCode[] { KeyCode.W, KeyCode.A, KeyCode.S, KeyCode.D, KeyCode.UpArrow, KeyCode.LeftArrow, KeyCode.DownArrow, KeyCode.RightArrow };
```

Comment:
Shouldn't this be an else?

Reply:
Technically yes, but it isn't needed since there is a return inside the if. It is basically an implicit else, should i add it anyway for the sake of readability?